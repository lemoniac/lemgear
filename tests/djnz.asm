	ld B, 10
	ld A, 0
	ld C, 1
l:
	add C
	djnz l

	halt
	
	ld A, 20
