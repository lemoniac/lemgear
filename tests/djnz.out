AF : 0A00
BC : 0001
